Libraries/Opc.Ua.Server/Subscription/MonitoredItem.cs

```diff
@@ -346,6 +346,13 @@ public Session Session
                     return m_session;
                 }
             }
+            set
+            {
+                lock(m_lock)
+                {
+                    m_session = value;
+                }
+            }
```

Comment:
it were better if monitored items uses a 'parent' to get the session

Reply:
Changed all code, that Monitored Item don't store Session directly anymore, it will read on demand from parent (subscription)